Examples/Basic/README.md

```diff
@@ -0,0 +1,48 @@
+# Swift OTel Basic Example
+
+Demonstrates how to get started with [Swift OTel](https://github.com/slashmo/swift-otel).
+
+## Overview
+
+This example contains an endless running service that keeps counting up a number after a randomized delay.
+
+Each increment is instrumented by a span, as shown in the [Jaeger](https://jaegertracing.io) screenshot below.
+
+![Screenshot of Jaeger displaying recorded spans from this example](tracing-counter-example.png)
+
+In addition to the exported spans, the example also logs every time the count is incremented. These log
+statements automatically include the trace/span ID of the current span:
+
+```sh
+2023-11-24T12:00:29+0100 info Counter : count=262 span-id=41d259b120114123 trace-id=08536b458fb95482a59fa15ee185fcaf [Example] Counted up.
```

Comment:
I wonder if also parent span id would be nice here? and in the default log metadata provider?

Reply:
> I wonder if also parent span id would be nice here?

Do you mean showcasing parent/child spans in this example?

> and in the default log metadata provider?

Yeah, that could be good when filtering log statements by a span ID to also get logs of its immediate children. I'm also still missing the `trace-flags` metadata value, which should be included according to the OTel spec.

Reply:
i didn't yet get to the metadata provider, was more wondering if it already exposes that ID as well -- it should I suppose :)